homeassistant/generated/config_flows_v2.py

```diff
@@ -0,0 +1,1418 @@
+            "name": "Environment Canada",
+        },
+        "epson": {
+            "name": "Epson",
+        },
+        "escea": {
+            "name": "Escea",
+        },
+        "esphome": {
+            "name": "ESPHome",
+        },
+        "evil_genius_labs": {
+            "name": "Evil Genius Labs",
+        },
+        "ezviz": {
+            "name": "Ezviz",
+        },
+        "faa_delays": {
+            "name": "FAA Delays",
+        },
+        "fibaro": {
+            "name": "Fibaro",
+        },
+        "filesize": {
+            "name": "File Size",
+            "translated_name": True,
+        },
+        "fireservicerota": {
+            "name": "FireServiceRota",
+        },
+        "fivem": {
+            "name": "FiveM",
+        },
+        "fjaraskupan": {
+            "name": "Fjäråskupan",
+        },
+        "flick_electric": {
+            "name": "Flick Electric",
+        },
+        "flipr": {
+            "name": "Flipr",
+        },
+        "flo": {
+            "name": "Flo",
+        },
+        "flume": {
+            "name": "Flume",
+        },
+        "flunearyou": {
+            "name": "Flu Near You",
+        },
+        "flux_led": {
+            "name": "Magic Home",
+        },
+        "forecast_solar": {
+            "name": "Forecast.Solar",
+        },
+        "forked_daapd": {
+            "name": "forked-daapd",
+        },
+        "foscam": {
+            "name": "Foscam",
+        },
+        "freebox": {
+            "name": "Freebox",
+        },
+        "freedompro": {
+            "name": "Freedompro",
+        },
+        "fritz": {
+            "name": "AVM FRITZ!Box Tools",
+        },
+        "fritzbox": {
+            "name": "AVM FRITZ!SmartHome",
+        },
+        "fritzbox_callmonitor": {
+            "name": "AVM FRITZ!Box Call Monitor",
+        },
+        "fronius": {
+            "name": "Fronius",
+        },
+        "fully_kiosk": {
+            "name": "Fully Kiosk Browser",
+        },
+        "garages_amsterdam": {
+            "name": "Garages Amsterdam",
+            "translated_name": True,
+        },
+        "gdacs": {
+            "name": "Global Disaster Alert and Coordination System (GDACS)",
+        },
+        "generic": {
+            "name": "Generic Camera",
+        },
+        "geocaching": {
+            "name": "Geocaching",
+        },
+        "geofency": {
+            "name": "Geofency",
+        },
+        "geonetnz_quakes": {
+            "name": "GeoNet NZ Quakes",
+        },
+        "geonetnz_volcano": {
+            "name": "GeoNet NZ Volcano",
+        },
+        "gios": {
+            "name": "GIOŚ",
+        },
+        "github": {
+            "name": "GitHub",
+        },
+        "glances": {
+            "name": "Glances",
+        },
+        "goalzero": {
+            "name": "Goal Zero Yeti",
+        },
+        "gogogate2": {
+            "name": "Gogogate2 and ismartgate",
```

Comment:
Shouldn't empty entries also have `"integrations": { "gogogate2": ... } }` or is that something we assume if omitted to be filled in ?

Reply:
I guess omitted works fine here.

Reply:
Yeah, I think we should omit it to keep the size of the index down.